src/pages/post-feed/Post.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from 'next/image';
+import { FacebookIcon, LinkedInIcon, TwitterIcon, EmailIcon, SandwichButton } from './icons';
+import { User, users } from './usersDummyData';
+
+export default function Post() {
```

Comment:
❌ 
this component is rending a **list of users** but it is called `Post` in **singular** form
change it

Reply:
Also, this component I changed already😢.

Reply:
I am seeing, and it didn't change
maybe you created another file and forgot to remove this one?